src/dashboard/src/templates/administration/sidebar.html

```diff
@@ -48,7 +49,11 @@
       <li class="{% active request url_user_list %}"><a href="{{ url_user_list }}">{% trans "Users" %}</a></li>
     {% endif %}
 
-    <li class="{% active request url_language %}"><a href="{{ url_language }}">{% trans "Language" %}</a></li>
+    <li class="{% active request handle_config %}"
+        ><a href="{{ handle_config }}">{% trans "Handle server config" %}</a></li>
```

Comment:
This `>` in a new line is pretty new to me, never seen this before in the Dashboard. Why are you doing it this way? Do we want to start forcing a 79col rule in HTML templates too from now on?